pow/src/main/java/tech/pegasys/teku/pow/fallback/strategy/FallbackStrategy.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.pow.fallback.strategy;
+
+public enum FallbackStrategy {
+  PRIORITY,
+  RANDOM,
+  ROUND_ROBIN
```

Comment:
I don't see a need for anything beyond priority here.  The main use-case will be people running their own Eth1 node but wanting to automatically fallback to infer if it's not yet in sync or goes down.

Reply:
Ok i will remove that